src/main/java/org/folio/qm/service/impl/ValidationServiceImpl.java

```diff
@@ -83,22 +86,43 @@ public void validateIdsMatch(QuickMarcEdit quickMarc, UUID parsedRecordId) {
   }
 
   @Override
-  public void validateMarcRecord(BaseMarcRecord marcRecord) {
+  public void validateMarcRecord(BaseMarcRecord marcRecord, boolean is001RequiredField) {
     if (marcRecord.getMarcFormat() != MarcFormat.HOLDINGS) {
       log.debug("validateMarcRecord:: validate a quickMarc record");
       var validatableRecord = converter.convert(marcRecord);
-      var validationIssues = validate(validatableRecord);
-      if (containsErrorSeverityType(validationIssues)) {
+      var validationIssues = getValidationIssuesWithRuleCode(validatableRecord);
```

Comment:
Will be better to do conversion after filtering. 
1. You get `ValidationError` objects.
2. Filter out everything that is not interested in specific case (001 required error, and warn errors)
3. Do a conversion to ValidationIssue

Reply:
In this case you don't need to use regex. So it could be replaced for a more elegant check.

Reply:
We can't avoid the regular expression because the ValidationError object doesn't contain the pure tag name, but only the "path" field with the tag name inside:
![image](https://github.com/user-attachments/assets/ad71eabc-55fa-4166-ad25-ad21ee2527bb)